FlowCal/excel_ui.py

```diff
@@ -212,6 +212,7 @@ def write_workbook(filename, table_list, column_width=None):
 
 def process_beads_table(beads_table,
                         instruments_table,
+                        get_transform_fxn_kwargs={},
```

Comment:
I think this should be the last parameter. This is such an edge case that I don't think changing the order of parameters is appropriate.